.github/workflows/executor-cd.yml

```diff
@@ -0,0 +1,33 @@
+name: CD
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+#  pull_request:
+# uncoment the line above to test CD push in a PR
+
+jobs:
+  push-executors:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get previous tag
+        id: previoustag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        with:
+          fallback: ""
+      - name: push to Hubble
+        id: push_exec
+        uses: jina-ai/action-hubble-push@v2
```

Comment:
As I think, cd.yml is for the push on the main branch and release.yml is for releasing a specific version. However, as Tadej pointed out, currently cd.yml updates the `latest` and `GIT_TAG` versions which I will change so that it only updates the `latest` version.